images/router/haproxy/conf/haproxy-config.template

```diff
@@ -199,9 +243,22 @@ frontend fe_no_sni
 
   # map to http backend
   # Search from most specific to general path (host case).
+  acl edge_http_backend base,map_beg(/var/lib/haproxy/conf/os_edge_http_be.map) -m found
+  use_backend be_edge_http_%[base,map_beg(/var/lib/haproxy/conf/os_edge_http_be.map)] if edge_http_backend
+
+{{ if matchPattern "true|TRUE" (env "ROUTER_ALLOW_WILDCARD_ROUTES" "")}}
```

Comment:
I would expect the expense to show up when switching among lots of routes, not repeatedly hitting a router serving few routes.